test/integration/singlecluster/tas/tas_test.go

```diff
@@ -356,6 +357,20 @@ var _ = ginkgo.Describe("Topology Aware Scheduling", ginkgo.Ordered, func() {
 				}
 			})
 
+			ginkgo.It("should expose the TopologyName in LocalQueue status", func() {
+				levels := slices.Map(topology.Spec.Levels, func(level *kueuealpha.TopologyLevel) string {
+					return level.NodeLabel
+				})
```

Comment:
IIUC we can use utiltas.Levels instead? Here and in the other test.

Reply:
Thanks!

Reply:
Done